src/v/redpanda/admin/server.cc

```diff
@@ -4347,6 +4348,14 @@ admin_server::delete_cloud_storage_lifecycle(
 ss::future<ss::json::json_return_type>
 admin_server::post_cloud_storage_cache_trim(
   std::unique_ptr<ss::http::request> req) {
+    co_await ss::smp::submit_to(ss::shard_id{0}, [this] {
```

Comment:
>Fixes UBSAN assertion

I can probably infer the UB here, but can you provide the assert or quickly summarize the issue in the commit message?